### ISSUE TITLE
service fabric NoneType error

### ISSUE THREAD

**Issue by zqxjjj** [Azure/azure-cli#12660]
## Describe the bug

**Command Name**
`az sf cluster create`

**Errors:**
```
'NoneType' object has no attribute 'get'
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 608, in execute
    raise ex
cli/core/commands/__init__.py, ln 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
...
cli/command_modules/servicefabric/custom.py, ln 1158, in _create_certificate
    cmd, cli_ctx, vault_resource_group_name, vault_name, location, enabled_for_deployment=True).result()
cli/command_modules/servicefabric/custom.py, ln 1667, in _create_keyvault
    permissions = Permissions(keys=[KeyPermissions.get,
AttributeError: 'NoneType' object has no attribute 'get'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az sf cluster create --location {} --resource-group {} --certificate-output-folder {} --certificate-password {} --certificate-subject-name {} --cluster-name {} --cluster-size {} --os {} --vault-name {} --vm-password {} --vm-user-name {} --vault-resource-group {}`

## Environment Summary
```
Linux-4.15.0-1071-azure-x86_64-with-debian-stretch-sid
Python 3.6.5

azure-cli 2.2.0

```

**Comment by yungezz** [comment]
thanks for reporting, we'll have a look

**Comment by a-santamaria** [comment]
thanks for reporting this its duplicate of #12470. it will be fix in the next release with pr #12731. Please use  Azure CLI 2.0.81 in the meantime.
